lib/workers/repository/process/index.spec.ts

```diff
@@ -120,5 +121,33 @@ describe('workers/repository/process/index', () => {
       });
       expect(lookup).toHaveBeenCalledTimes(0);
     });
+
+    it('finds baseBranches via regular expressions', async () => {
+      extract.mockResolvedValue({} as never);
+      config.baseBranches = ['/^release\\/.*/', 'dev', '!/^pre-release\\/.*/'];
+      git.getBranchList.mockReturnValue([
+        'dev',
+        'pre-release/v0',
+        'release/v1',
+        'release/v2',
+        'some-other',
+      ]);
+      git.branchExists.mockReturnValue(true);
+      const res = await extractDependencies(config);
+      expect(res).toEqual({
+        branchList: [undefined, undefined],
```

Comment:
we should filter undefined in future PR

Reply:
I think we could also drop it from this test. I've replicated these lines from the other tests but they are not essential for what the test is supposed to check.

Reply:
ok, the undefined come from mocked functions which don't have a return value set?